routing_model/index.html

```diff
@@ -132,8 +141,58 @@ <h2>Node as a member of a section</h2>
 </div>
 
   <h1>Destination section</h1>
-  Receiving a node that is being relocated.
+  <p>As a member of a section, our section will sometimes receive a node that is being relocated. These  diagrams are from the point of view of one of the nodes in the section, doing its part to handle the node that is trying to rellocate to this section.
+  </p>
   <h2>Processing a candidate trying to join our section</h2>
+  <p>This flow represents what we do when a candidate contacts us to join our section.<br>
+  The process starts as we receive an ExpectCandidate RPC from this node.<br>
+  We vote for it in PARSEC to be sure all members of the section process it in the same order.<br>
+  Once it reaches consensus, we are ready to process that candidate by letting them perform the resource_proof (see AcceptAsCandidate).<br>
+  There are some subtleties, such as the fact that we only want to process one candidate at a time, but this is the general idea.
+  </p>
+  <button class="collapsible">RPC: ExpectCandidate</button>
+  <div class="content">
+    <p>We receive this RPC from a node that wants to join our section.<br>
+    They aren't connected to us, yet; so the RPC reaches us through their proxy node.<br>
```

Comment:
The node is not communicating with us yet, only once we sent RelocateResponse RPC to the originating section.